src/components/Form.jsx

```diff
@@ -12,24 +13,126 @@ class Form extends Component {
             clientEmail: '',
             clientHouseAge: '',
             clientAddress: '',
-            problem: false,
             knowProb: 'No',
             clientQuestion: '',
-            problemDesc: ''
+            problemDesc: '',
+            clientZip: '',
+            clientCare: '',
+            clientPay: '',
+            anotherProb: 'No',
+            waterTesting: 'No',
+            formErrors: {
+                clientName: '',
+                clientEmail: '',
+                clientHouseAge: '',
+                clientAddress: '',
+                knowProb: '',
+                clientQuestion: '',
+                problemDesc: '',
+                clientZip: '',
+                clientCare: '',
+                clientPay: '',
+                anotherProb: '',
+                waterTesting: ''
+            },
+            clientNameValid: '',
+            clientEmailValid: '',
+            clientHouseAgeValid: '',
+            clientAddressValid: '',
+            clientKnowProbValid: '',
+            clientQuestionValid: '',
+            clientProblemDescValid: '',
+            clientZipValid: '',
+            clientCareValid: '',
+            clientPayValid: '',
+            anotherProbValid: '',
+            waterTestValid: ''
           };
         this.handleChange=this.handleChange.bind(this);
         this.handleSubmit=this.handleSubmit.bind(this);
     }
 
+    validateField(fieldName, value) {
+        let fieldValidationErrors = this.state.formErrors;
+        let clientNameValid = this.state.clientNameValid;
+        let clientEmailValid = this.state.clientEmailValid;
+        let clientHouseAgeValid = this.state.clientHouseAgeValid;
+        let clientAddressValid = this.state.clientAddressValid;
+        let clientKnowProbValid = this.state.clientKnowProbValid;
```

Comment:
I understand setting all the variables up in case we do any form validation for things other than what he asked for and or you were doing testing during development. Before merging into dev I would refactor this to only define variables used in our validation just because it would limit the wall of text we have going on here.